src/zenml/cli/stack_recipes.py

```diff
@@ -60,159 +53,6 @@
     f'`pip install "zenml[stacks]=={zenml.__version__}"`.'
 )
 
-try:
```

Comment:
what happened here with this mega-deletion?

Reply:
we shifted the stack recipe service to a separate class under zenml.recipes